### PR TITLE
Preparing new version

### PR DESCRIPTION
**Pull request by vincentadam87**
The current code
* fixes the scipy dependency issues
* removes unknown tag from pyproject.toml which prevented installation

### CODE REVIEW

**Review by uri-granta — COMMENTED** [review]
Shouldn't the Version be updated in develop first?

File: pyproject.toml [hunk 1]
```diff
@@ -91,4 +94,4 @@ line_length = 95
 [build-system]
 requires = ["poetry>=0.12", "cmake", "setuptools"] #, "tensorflow>=2.8.0,<2.9.0", "cmake"]
 build-backend = "poetry.masonry.api"
-flags = ["-DCMAKE_CXX_STANDARD=14"]
+# flags = ["-DCMAKE_CXX_STANDARD=14"] (removing because flags not a recognized property)
```
Comment (awav) [inline pyproject.toml:97]: I think that should be added to the list of `cmake` flags: https://github.com/secondmind-labs/banded_matrices/blob/3a765fc052544e3be75c425a951c1c417f4d6150/build.py#L65. I'm not quite sure how did it work before. Also, I would bump the standard to 17 or even 20 if possible.

**Review by awav — CHANGES_REQUESTED** [review]
Please, address one my comment. Otherwise, LGTM.

**Comment by awav** [comment]
Also, @uri-granta was right about the version :)

**Pull request merged**